### PR TITLE
(EZ-78) Docs for bootstrap work

### PR DESCRIPTION
**Pull request by jpinsonault**
This adds docs describing how to use the new ezbake settings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,10 +158,17 @@ repository.
...
+By default, in the final packages produced by an ezbake build, there will be a "config" directory
 (usually `/etc/puppetlabs/<ezbake-project-name>`) that contains all of the final
 configuration files for the ezbake application.
 
+If the `:bootstrap-source` setting is set to `:services-d`, there will also be an additional
+configuration directory under the installation directory:
+`/opt/puppetlabs/server/apps/{project-name}/config/`
+This directory will then be used to hold a `services.d/` directory that contains bootstrap files.
+Usually this will be for bootstrap entries the application could not run without, and which should
+not be modified by the user. See [Bootstrap Files](#bootstrap_files)
```
Comment (cprice404) [inline README.md]: I think I might add a subsection markdown header before this paragraph, called something like "bootstrap-source".  Then explicitly mention the two legal values for the enum here.  It's fine to keep this paragraph brief and defer the more detailed explanation to the later section that you added, but I think you should at least call out the legal values for the setting.

File: README.md [hunk 2]
```diff
@@ -206,6 +218,36 @@ The rule of thumb is:
...
   `config/conf.d/web-routes.conf`.
 
+### Bootstrap Files
+There are two ways of specifying a set of services to bootstrap your TK application with:
+
+1. A single `bootstrap.cfg` file in your project's `:config-dir` directory
+2. Two `services.d` directories which can contain any number of `*.cfg` files that will be
+   merged together
+
+The default is to use a single `bootstrap.cfg` directly under your `:config-dir`. Setting
```
Comment (cprice404) [inline README.md]: Would also mention the default value of the `:bootstrap-source` setting here, in case anyone prefers to be explicit in their config.

**Comment by cprice404** [comment]
+1 other than my comments above.

**Comment by jpinsonault** [comment]
@cprice404 Addressed your comments, how's that look?

**Pull request merged**